packages/amplify-util-mock/src/api/api.ts

```diff
@@ -383,12 +386,18 @@ export class APITest {
     if (!name) {
       throw new AmplifyFault('MockProcessFault', {
         message: 'No AppSync API is added to the project',
-        link: AMPLIFY_SUPPORT_DOCS.CLI_GRAPHQL_TROUBLESHOOTING.url,
+        resolution: `Use 'amplify add api' in the root of your app directory to create an API. Today, we only support mocking of GraphQL API`,
```

Comment:
technically both AmplifyError & AmplifyFault can be used here to throw an exception. But if you look here in the amplify-exception.ts file where the type of error MockProcessFault is of type AmplifyFaultType. for reference:

[amplify-cli/packages/amplify-cli-core/src/errors/amplify-exception.ts](https://github.com/aws-amplify/amplify-cli/blob/8e44b0da1af532a90a8e39c880f7d7a0db197e81/packages/amplify-cli-core/src/errors/amplify-exception.ts#L200)

Line 200 in [8e44b0d](https://github.com/aws-amplify/amplify-cli/commit/8e44b0da1af532a90a8e39c880f7d7a0db197e81)

 | 'MockProcessFault'
 
reference comment- https://github.com/aws-amplify/amplify-cli/pull/12524#discussion_r1179443883

Reply:
Amplify Fault is a problem in the source code whereas AmplifyError is a user error. It's important that we get it right for the telemetry. This seems like the customer is missing something so I would classify this as a AmplifyError.